install/tools/man/ipa-migrate.1

```diff
@@ -0,0 +1,111 @@
+In this mode, SIDs & DNA ranges are not migrated, and DNA attributes are reset
+.SH "COMMANDS"
+.TP
+\fB\-v\fR, \fB\-\-verbose\fR
+Use verbose output while running hte migration tool.
+.TP
+\fB\-v\fR, \fB\-\-verbose\fR
+Use verbose output while running hte migration tool.
+.TP
+\fB\-\-hostname=HOSTNAME\fR
+The host name of the remote IPA server that is being migrated from.
+.TP
+\fB\-\-bind\-dn=BIND_DN\fR
+The Bind DN (Distinguished Name) or an LDAP entry to bind to the remote IPA server with.
+Typically this is "cn=directory manager", but it could be any entry that has
+access to read the userPassword attribute.
+.TP
+\fB\-\-bind\-pw=PASSWORD\fR
+The password for the Bind DN that is authenticating against the remote IPA server.  If
+a password is not provided then the tool with prompt for the password if needed.
+.TP
+\fB\-\-bind\-pw\-file=FILE_PATH\fR
+Path to a file containing the password for the Bind DN.
+.TP
+\fB\-\-cacertfile\-file=FILE_PATH\fR
+Path to a file containing a CA Certificate that the remote server trusts
+.TP
+\fB\-\-log\-file=FILE_PATH\fR
+Path to a file containing the migration log.  By default the tool will use \fI/var/log/ipa-migrate.log\fR
+.TP
+\fB\-\-dryrun\fR
+Go through the migration process but do not write and data to the new IPA server.
+.TP
+\fB\-\-dryrun-ldif-file=FILE_PATH\fR
+Go through the migration process but do not write and data to the new IPA server. However, write the
+migration operations to an LDIF file which can be apllied later or reused for multiple migrations.
+.TP
+\fB\-F\fR, \fB\-\-force\fR
+Ignore any errors and continue to proceed with migration effort.
+.TP
+\fB\-\-version\fR
+Display the version of the migration tool.
+.TP
+\fB\-q\fR, \fB\-\-quiet\fR
+Only log errors during the migration process.
+.TP
+\fB\-\-skip\-dns\fR
+Do not migrate DNS records
+.TP
+\fB\-\-skip\-schema\fR
+Do not migrate the database schema
+.TP
+\fB\-\-skip\-config\fR
+Do not migrate the database configuration (dse.ldif/cn=config)
+.TP
+\fB\-\-schema\-overwrite\fR
+Overwrite existing schema definitions.  By default duplicate schema is skipped.
+.TP
+\fB\-\-subtree=DN\fR
+Specifies a custom database subtree that should be included in the migration.
+This is only needed if non-default subtrees/branches were added to the database
+outside of IDM.
+.TP
+\fB\-\-db\-ldif=FILE_PATH\fR
+LDIF file containing the entire backend. If omitted the tool will query the remote IPA server.
+.TP
+\fB\-\-schema\-ldif=FILE_PATH\fR
+LDIF file containing the schema. If omitted the tool will query the remote IPA server.
+.TP
+\fB\-\-config\-ldif=FILE_PATH\fR
+LDIF file containing the entire "cn=config" DIT. If omitted the tool will query the remote IPA server.
+.TP
+\fB\-n\fR, \fB\-\-no\-prompt\fR
+Do not prompt for confirmation before starting migration.  Use at your own risk!
+.TP
+
+.SH "EXIT STATUS"
+0 if the command was successful
+
+1 if an error occurred
+
+2 if the host is not a IPA server, or the IPA server installation is faulty
```

Comment:
Many of the short options are not documented in the man page. For example -h, -e. There are more.

-v is documented twice, typo: hte

Extra -file on --cacertfile-file

What does --dryrun-ldif-file do if --dryrun is not present? The implication in the cli help is that it doesn't do a dry-run

I would prefer --dry-run as that is common to other commands (e.g. git commit)